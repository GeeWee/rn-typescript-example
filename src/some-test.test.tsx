import 'jest';
import React from 'react';
import * as renderer from 'react-test-renderer';
import { Text, View } from 'react-native';

interface SomeInterface {
	foo: string,
	bar: number
}

describe('Basic breakpoints and sourcemaps', () => {
	it('Should show you the value of the variables the whole way through', () => {
		const x = 2;
		let y = 3;
		y = 3;
		y = 4;
		y = 5;
		y = 6;
		y = 3;
		let z = x + y;
		//debugger;
		const foo = 34;
		expect(z).toEqual(5);
	});
});

const SomeComponent = () => {
	return (
		<View>
			<Text>
				Hello world
			</Text>
		</View>
	)
};

describe('Some component', () => {
	it('Should render via react test renderer', () => {
		const jsonObject = renderer.create(<SomeComponent />).toJSON();
		expect(jsonObject).toMatchSnapshot();
	});
});