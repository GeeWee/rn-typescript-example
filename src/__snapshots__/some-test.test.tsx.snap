// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Some component Should render via react test renderer 1`] = `
<View>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  >
    Hello world
  </Text>
</View>
`;
